Terminal.Gui/Windows/FileDialog.cs

```diff
@@ -116,11 +116,17 @@ protected override void Dispose (bool disposing)
 
 		void Watcher_Error (object sender, ErrorEventArgs e)
 		{
+			if (Application.MainLoop == null)
```

Comment:
1) This fixes an issue not related to this PR. Please remove from this PR and do another one.
2) If statement needs braces.

Reply:
@tig this change is already belong to the develop branch and if it's really need to change also would be need to change this:

![imagem](https://user-images.githubusercontent.com/13117724/205513542-098f6a54-6715-4f88-bf66-8be065e562a5.png)

Do you really think it worth it?